t0/seqio_tasks/experiment_D4.csv

```diff
@@ -0,0 +1,73 @@
+HF_name,subset,task_by_convention,format,comment,seed_paper,do_train,do_eval,train_size,adjusted_train_size,metric,multiple correct answer,Paper link,non_linguistic_knowledge,Imported Task Name,imported category,input_length,_human_skill,Domain,Reference,done

```

Comment:
we are only using a subset of the columns:
- HF_name
- subset
- do_train
- do_eval
- task_by_convention
- train_size

I suggest we remove the others for readability as they were mainly useful when we were iterating on the training/evaluation mixtures